tests/Teams/ActionBasedMessagingExtension-FetchTask/AdaptiveCardHelper.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using AdaptiveCards;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public static class AdaptiveCardHelper
+    {
+        public static SubmitExampleData GetSubmitExampleDataFromAction(MessagingExtensionAction action)
+        {
+            var activityPreview = action.BotActivityPreview[0];
+            var attachmentContent = activityPreview.Attachments[0].Content;
+            var previewedCard = JsonConvert.DeserializeObject<AdaptiveCard>(attachmentContent.ToString(), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
+            string userText = (previewedCard.Body[1] as AdaptiveTextBlock).Text;
+            var choiceSet = previewedCard.Body[3] as AdaptiveChoiceSetInput;
+
+            return new SubmitExampleData()
+            {
+                Question = userText,
+                MultiSelect = choiceSet.IsMultiSelect ? "true" : "false",
+                Option1 = choiceSet.Choices[0].Title,
+                Option2 = choiceSet.Choices[1].Title,
+                Option3 = choiceSet.Choices[2].Title,
+            };
+        }
+
+        public static MessagingExtensionActionResponse GetTaskModuleAdaptiveCardResponse(string userText = null, bool isMultiSelect = true, string option1 = null, string option2 = null, string option3 = null)
```

Comment:
This seems like an odd function name?
It doesn't "Get" anything, it seems to "Create" a MessagingExtensionActionResponse.

Reply:
Changed to CreateTaskModuleAdaptiveCardResponse